src/assets/index.css

```diff
@@ -107,7 +107,73 @@ padding: 2%;
 width: 1000px !important;
 display:none;
 }
-/* ----------------------- */
-#result_container{
-  padding-top: 150px;
+
+/* This is the CSS part used in Input form */
+
+#upload {
+  opacity: 0;
+}
+
```

Comment:
According to me there are some unwanted css, remove that (#upload, #upload-label ....) check once, and remove it.